villain/externs.rkt

```diff
@@ -30,6 +30,8 @@
     [(App f es)
      (append (externs-f f)
              (externs-es es))]
+    [(LCall e es)
```

Comment:
This should replace the `Call` form.

Reply:
Yes, thanks!  I kept it for the compilation of the library function definitions.  If we do the compilation of the definitions of these functions via `(define (stdlibs program) (Letrec fs ls program)` and then `(compile (stdlibs program))`, we can remove the `(App f es)` struct.